### ISSUE TITLE
OpenMod Command Line issue or sorts. Un-Resolved as to the real issue, Please Help!

### ISSUE THREAD

**Issue by ZanyISP** [openmod/openmod#618]
## The Main issue in layman's terms is that after trying to utilize both RocketMod and OpenMod Roles/Permissions System (by changing it in openmod.unturned.yaml) and Multiple Edits of the Permissions and re-installation of the plugins I find it still telling me command not found for simple things like /jump, /destroy, /refuel, and more. Here is some More info...

After Running the Command In Game I am met with these two output results.
**In Game Result**
![image](https://user-images.githubusercontent.com/33303170/180645329-70e108b0-52ef-49a6-89d0-c26f4034fe82.png)
**Console Output Result After Running In-Game Command**
![image](https://user-images.githubusercontent.com/33303170/180645361-578954ac-49c5-4da5-8fa0-b07e96129eb0.png)

## Here is a Copy of my openmod.roles.yaml for further information for those who know more than me.
I understand it is messy and complete dog water.
```
roles:
- id: default
  parents: []
  permissions:
  - OpenMod.Core:help
  - p
  - rocket
  - Kr4ken.NewEssentials:reply
  - Kr4ken.NewEssentials:tell
  - donate
  - discord
  - rules
  - report
  - Kr4ken.NewEssentials:tpa.accept
  displayName: Default
  data: {}
  isAutoAssigned: true
- id: vip
  displayName: VIP
  priority: 1
  parents:
  - default
  permissions:
  - Kr4ken.NewEssentials:kits.vip
  - OpenMod.Core:help
  - p
  - rocket
  - Kr4ken.NewEssentials:reply
  - Kr4ken.NewEssentials:tell
  - donate
  - discord
  - rules
  - report
  data: {}
- id: vipplus
  displayName: VIP+
  priority: 1
  parents:
  - default
  permissions:
  - Kr4ken.NewEssentials:kits.vipplus
  - OpenMod.Core:help
  - p
  - rocket
  - Kr4ken.NewEssentials:reply
  - Kr4ken.NewEssentials:tell
  - donate
  - discord
  - rules
  - report
  data: {}
- id: mvp
  displayName: MVP
  priority: 1
  parents:
  - default
  permissions:
  - Kr4ken.NewEssentials:kits.mvp
  - OpenMod.Core:help
  - p
  - rocket
  - Kr4ken.NewEssentials:reply
  - Kr4ken.NewEssentials:tell
  - donate
  - discord
  - rules
  - report
  data: {}
- id: mvpplus
  displayName: MVP+
  priority: 1
  parents:
  - default
  permissions:
  - Kr4ken.NewEssentials:kits.mvpplus
  - OpenMod.Core:help
  - p
  - rocket
  - Kr4ken.NewEssentials:reply
  - Kr4ken.NewEssentials:tell
  - donate
  - discord
  - rules
  - report
  data: {}
- id: srmod
  displayName: SrMod
  priority: 1
  parents:
  - default
  permissions:
  - OpenMod.Core:help
  - p
  - rocket
  - Kr4ken.NewEssentials:reply
  - Kr4ken.NewEssentials:tell
  - donate
  - discord
  - rules
  - report
  - heal
  - effect
  - heal
  - v
  - compass
  - kits
  - Kr4ken.NewEssentials:tpa
  - Kr4ken.NewEssentials:tpa.send
  - Kr4ken.NewEssentials:tpa.accept
  - Kr4ken.NewEssentials:tpa.deny
  - Kr4ken.NewEssentials:tpa.cancel
  - home
  - Kr4ken.NewEssentials:back
  - Kr4ken.NewEssentials:tp
  - teleport
  - tphere
  - vanish
  - Kr4ken.NewEssentials:vanish
  - Feli.AdminTools:commands.handdestroyer
  - Feli.AdminTools:commands.whois
  data: {}
- id: jrmodmod
  displayName: JrMod
  priority: 1
  parents:
  - default
  permissions:
  - OpenMod.Core:help
  - p
  - rocket
  - Kr4ken.NewEssentials:reply
  - Kr4ken.NewEssentials:tell
  - donate
  - discord
  - rules
  - report
  - heal
  - effect
  - heal
  - v
  - compass
  - kits
  - Kr4ken.NewEssentials:tpa
  - Kr4ken.NewEssentials:tpa.send
  - Kr4ken.NewEssentials:tpa.accept
  - Kr4ken.NewEssentials:tpa.deny
  - Kr4ken.NewEssentials:tpa.cancel
  - home
  - Kr4ken.NewEssentials:back
  - Kr4ken.NewEssentials:tp
  - Feli.AdminTools:commands.door
  - Feli.AdminTools:commands.handdestroyer
  - Feli.AdminTools:commands.storage
  - Feli.AdminTools:commands.whois
  data: {}
- id: admin
  displayName: Admin
  priority: 1
  parents:
  - default
  permissions:
  - OpenMod.Core:help
  - p
  - rocket
  - Kr4ken.NewEssentials:reply
  - Kr4ken.NewEssentials:tell
  - donate
  - discord
  - rules
  - report
  - effect
  - heal
  - v
  - compass
  - kits
  - Kr4ken.NewEssentials:tpa
  - Kr4ken.NewEssentials:tpa.send
  - Kr4ken.NewEssentials:tpa.accept
  - Kr4ken.NewEssentials:tpa.deny
  - Kr4ken.NewEssentials:tpa.cancel
  - home
  - Kr4ken.NewEssentials:warp
  - Kr4ken.NewEssentials:warps
  - Kr4ken.NewEssentials:back
  - Kr4ken.NewEssentials:maxskills
  - Kr4ken.NewEssentials:tp
  - teleport
  - tphere
  - vanish
  - god
  - clonei
  - dropammo
  - ammo
  - spy
  - jump
  - refuel
  - Kr4ken.NewEssentials:vanish
  - Kr4ken.NewEssentials:god
  - Kr4ken.NewEssentials:clonei
  - Kr4ken.NewEssentials:refuel
  - Kr4ken.NewEssentials:dropammo
  - Kr4ken.NewEssentials:ammo
  - Kr4ken.NewEssentials:jump
  - Feli.AdminTools:commands.clear
  - Feli.AdminTools:commands.clearinventory
  - Feli.AdminTools:commands.door
  - Feli.AdminTools:commands.handdestroyer
  - Feli.AdminTools:commands.salvage
  - Feli.AdminTools:commands.storage
  - Feli.AdminTools:commands.whois
  - EasyAmmo:ammo
  - EasyAmmo:dropammo
  data: {}
- id: owner
  displayName: Owner
  priority: 1
  parents:
  - default
  permissions:
  - OpenMod.Core:help
  - p
  - rocket
  - Kr4ken.NewEssentials:reply
  - Kr4ken.NewEssentials:tell
  - donate
  - discord
  - rules
  - report
  - effect
  - heal
  - v
  - compass
  - kits
  - Kr4ken.NewEssentials:tpa
  - Kr4ken.NewEssentials:tpa.send
  - Kr4ken.NewEssentials:tpa.accept
  - Kr4ken.NewEssentials:tpa.deny
  - Kr4ken.NewEssentials:tpa.cancel
  - home
  - Kr4ken.NewEssentials:warp
  - Kr4ken.NewEssentials:warps
  - Kr4ken.NewEssentials:back
  - Kr4ken.NewEssentials:maxskills
  - Kr4ken.NewEssentials:tp
  - teleport
  - tphere
  - vanish
  - god
  - clonei
  - dropammo
  - ammo
  - spy
  - jump
  - refuel
  - Kr4ken.NewEssentials:vanish
  - Kr4ken.NewEssentials:god
  - Kr4ken.NewEssentials:clonei
  - Kr4ken.NewEssentials:refuel
  - Kr4ken.NewEssentials:dropammo
  - Kr4ken.NewEssentials:ammo
  - Kr4ken.NewEssentials:jump
  - Feli.AdminTools:commands.clear
  - Feli.AdminTools:commands.clearinventory
  - Feli.AdminTools:commands.door
  - Feli.AdminTools:commands.fly
  - Feli.AdminTools:commands.handdestroyer
  - Feli.AdminTools:commands.salvage
  - Feli.AdminTools:commands.speed
  - Feli.AdminTools:commands.storage
  - Feli.AdminTools:commands.whois
  - EasyAmmo:ammo
  - EasyAmmo:dropammo
  - Kr4ken.NewEssentials:command.jump
  - Kr4ken.NewEssentials:jump
  - Kr4ken.NewEssentials:command.back
  data: {}
- id: hidden
  displayName: hidden
  priority: 1
  parents:
  - default
  permissions:
  - OpenMod.Core:help
  - p
  - rocket
  - Kr4ken.NewEssentials:reply
  - Kr4ken.NewEssentials:tell
  - donate
  - discord
  - rules
  - report
  - effect
  - heal
  - v
  - compass
  - kits
  - Kr4ken.NewEssentials:tpa
  - Kr4ken.NewEssentials:tpa.send
  - Kr4ken.NewEssentials:tpa.accept
  - Kr4ken.NewEssentials:tpa.deny
  - Kr4ken.NewEssentials:tpa.cancel
  - home
  - Kr4ken.NewEssentials:warp
  - Kr4ken.NewEssentials:warps
  - Kr4ken.NewEssentials:back
  - Kr4ken.NewEssentials:maxskills
  - Kr4ken.NewEssentials:tp
  - teleport
  - tphere
  - vanish
  - god
  - clonei
  - dropammo
  - ammo
  - spy
  - jump
  - refuel
  - Kr4ken.NewEssentials:vanish
  - Kr4ken.NewEssentials:god
  - Kr4ken.NewEssentials:clonei
  - Kr4ken.NewEssentials:refuel
  - Kr4ken.NewEssentials:dropammo
  - Kr4ken.NewEssentials:ammo
  - Kr4ken.NewEssentials:jump
  - Feli.AdminTools:command.clear
  - Feli.AdminTools:command.clearinventory
  - Feli.AdminTools:command.door
  - Feli.AdminTools:command.fly
  - Feli.AdminTools:command.handdestroyer
  - Feli.AdminTools:command.salvage
  - Feli.AdminTools:command.speed
  - Feli.AdminTools:command.storage
  - Feli.AdminTools:command.whois
  - EasyAmmo:ammo
  - EasyAmmo:dropammo
  - Kr4ken.NewEssentials:command.jump
  - Kr4ken.NewEssentials:jump
  - Kr4ken.NewEssentials:command.back
  data: {}
```
## Would love to figure this out so I can actually get the permissions and commands working, Thanks!

**Comment by ZanyISP** [comment]
Before Anyone Says anything, yes I already Fixed the Kr4ken.NewEssentials to just NewEssentials as its supposed to be, still not fixed.

**Comment by IAmSilK** [comment]
@ZanyISP try adding `RocketMod:` to the beginning of all your Rocket permissions and see if that resolves the issue.

**Comment by Trojaner** [comment]
Closing due inactivity
